.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
I've tested this and it isn't being removed, at least in npm version `1.4.28`. Do you still want me to take this out of `.gitignore`?